rfc/rfc004-package-log.md

```diff
@@ -12,8 +12,21 @@ http://peter.bourgon.org/go-kit/#package-log
 
 ## Scope
 
-To be defined.
+* Log SHALL be drop in compatible with the standard library logger.
+* Log SHALL use key / value pairs for structured data.
```

Comment:
Agreed. Can you think of a use case for allowing per-log-call key/value pairs? e.g. `log.InfoWith("something happened: %d", count, "specialkey", "specialvalue")`